src/BBT.StructureTools/Convert/Strategy/IOperationCreateToOneHistWithCondition.cs

```diff
@@ -8,6 +8,14 @@
     /// <summary>
     /// Strategy to convert entities with a <c>ToMany</c> relationship.
     /// See <see cref="IConvertOperation{TSource,TTarget}"/>.
+    /// <typeparam name="TSource">See link above.</typeparam>
```

Comment:
Please keep in mind that we're generating a documentation from this, therefore it would make sense to write something here which afterwards resolves to an actual link. Applies also to other instances.

Reply:
See issue #66.